UKHomeOffice/sgmr-service
# SPCR-337 Remove unneeded duplicate ids

## Description

There were duplicate id's set on the div and the input elements on the edit account form
Removed the id's from the divs as they were not needed

## To test
- go to /account
- go to edit
- open AXE tools in devtools (needs chrome plugin)
- run scan
- > there should be no errors related to duplicate ids (there may be other errors that will be resolved in other tickets)


## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
